diff -ru pfstools-1.8.1.orig/src/filter/pfspanoramic.cpp pfstools-1.8.1/src/filter/pfspanoramic.cpp
--- pfstools-1.8.1.orig/src/filter/pfspanoramic.cpp	2009-10-29 11:37:25.000000000 +0100
+++ pfstools-1.8.1/src/filter/pfspanoramic.cpp	2009-10-29 11:49:45.000000000 +0100
@@ -195,7 +195,7 @@
     {
       char *opts;
       
-      if(opts = strchr(name, '/'))
+      if(opts = strchr((char*)name, '/'))
       {
         *opts++ = '\0';
       }
